runtime/src/lib.rs

```diff
@@ -552,6 +555,12 @@ parameter_types! {
         reserve_percent: Percent::from_percent(80),
     };
 	pub const ExchangeRate: Balance = 200_000_000_000_000_000; // 1 euro cent = 0.2 LGNT
+	pub const CertificateFee: u64 = 4_000_000_000_000_000; // 0.004 LGNT
+    pub const CertificateFeeDistributionKey: DistributionKey = DistributionKey {
```

Comment:
Wrong indentation.

Reply:
Fixed. I have also added the same `.editoconfig` file as for the node.